src/Aspire.Hosting/Utils/PasswordGenerator.cs

```diff
@@ -49,4 +50,20 @@ public static string GeneratePassword(int lowerCase, int upperCase, int digit, i
 
         return new string(chars);
     }
+
+    /// <summary>
+    /// Creates a random string of upper and lower case letters.
+    /// </summary>
+    public static string GenerateRandomLettersValue(int minLength)
+    {
+        ArgumentOutOfRangeException.ThrowIfNegative(minLength);
+        ArgumentOutOfRangeException.ThrowIfGreaterThan(minLength, 128);
+
+        Span<char> chars = stackalloc char[minLength];
+
+        RandomNumberGenerator.GetItems(LettersChars, chars);
+        RandomNumberGenerator.Shuffle(chars);
```

Comment:
nit, since you're only putting letters in, no shuffling should be required. but no harm leaving it for clarity, I guess.

Reply:
Good point. I can remove this line.

Although, once we decide on https://github.com/Azure/azure-dev/issues/3462 I believe this code is going to be refactored or basically just go away and a version of the above method will be used.